stellar-contract-env-common/src/env.rs

```diff
@@ -60,97 +60,17 @@ macro_rules! call_macro_with_all_host_functions {
             // argument list of comma-separated arg:type pairs
 
             mod context "x" {
-                {"$_", fn log_value(v:RawVal) -> RawVal }
-                {"$0", fn get_last_operation_result() -> RawVal }
-                {"$1", fn obj_cmp(a:RawVal, b:RawVal) -> i64 }
+                {"$_", fn obj_cmp(a:RawVal, b:RawVal) -> i64 }
```

Comment:
Should `a` and `b` also be `Object` type?

Reply:
Oh, yes, probbaly so. Good catch.